docs/HowToArchiveJournal.md

```diff
@@ -17,13 +17,134 @@ For more info on NuoDB database journaling, please consult the official [NuoDB d
 
 Since the `journal` has to write all commits durably to disk, the speed of the disk directly influences the commit latency.
 To achieve the best performance, NuoDB recommends placing the `journal` on the fastest disk available.
```

Comment:
Say fastest in what respect - for archive we specify throughput

Reply:
I believe that in the past you suggested `latency`?

Reply:
Yes we said journal is more latency sensitive and archive more throughput sensitive